### PR TITLE
Fix snapshotLinkedStylesheets undefined check

### PR DESCRIPTION
**Pull request by alexivanov**
Fixes case where falsy snapshotLinkedStylesheets value isn't set correctly.

### CODE REVIEW

File: packages/recorder-loader/src/loader.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export const loadAndStartRecorder: (options: LoaderOptions) => Promise<void> = (
       window["METICULOUS_APP_COMMIT_HASH"] = options.commitHash;
     }
 
-    if (options.snapshotLinkedStylesheets) {
+    if (typeof options.snapshotLinkedStylesheets !== "undefined") {
```
[the remark this answers is not in the document]
Comment (Que3216) [reply packages/recorder-loader/src/loader.ts:41]: (and do we need to make similar changes to the other if statements?)

[the remark this answers is not in the document]
Comment (alexivanov) [reply packages/recorder-loader/src/loader.ts:41]: >Don't we want this to resolve to false if snapshotLinkedStylesheets is false?
We do.

Within the recording snippet this setting defaults to `true` if unset. So a falsy `snapshotLinkedStylesheets` values won't propagate downwards to the snippet config.

[the remark this answers is not in the document]
Comment (alexivanov) [reply packages/recorder-loader/src/loader.ts]: > (and do we need to make similar changes to the other if statements?)

yeah we can, will update.

[the remark this answers is not in the document]
Comment (Que3216) [reply packages/recorder-loader/src/loader.ts]: Ah got it, yeah this makes sense then.

Also, can you also just do `options.snapshotLinkedStylesheets !== undefined`?

If you're worried about `options` being undefined you can change:

```
export const loadAndStartRecorder: (options: LoaderOptions)
```

To:

```
export const loadAndStartRecorder: (options: LoaderOptions = {})
```

**Review by Que3216 — APPROVED** [review]

**Pull request merged**